.pre-commit-config.yaml

```diff
@@ -91,3 +91,22 @@ repos:
     - id: add-license-headers
       args:
       - --start_year=2021
+
+- repo: local
+  hooks:
+    - id: check-vulnerabilities-dependencies
```

Comment:
I understand the need/interest to have it as part of the pre-commit stages but, in order to properly set up the tool and have it report them in case they appear, it is necessary that the check-vulnerabilities GitHub action is also set up

Reply:
@seanpearsonuk Please share your views. Thank you.